scripts/profile.sh

```diff
@@ -3,8 +3,9 @@ set -e
 TRACEFILE=data/profile.sqlite
 TRACELOG=data/profile.txt
 TRACEPYPROFLOG=data/profile.pyprof.txt
-
-nvprof -s -f -o $TRACEFILE --profile-from-start off -- python3 benchmark.py --profile-cuda $@ &> $TRACELOG
+LOGDEST=cudnn_log.txt
+CUBLAS_LOGDEST=cublas_log.txt
+CUDNN_LOGDEST_DBG=$LOGDEST CUBLAS_LOGDEST_DBG=$CUBLAS_LOGDEST nvprof -s -f -o $TRACEFILE --profile-from-start off -- python3 benchmark.py --profile-cuda $@ &> $TRACELOG
```

Comment:
also it's likely not very useful to profile with CUDNN/CUBLAS logs enabled, it can skew the timings, so don't do that

Reply:
done